package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bare-module",
-  "version": "4.8.2",
+  "version": "4.8.3",
```

Comment:
```suggestion
  "version": "4.8.2",
```

Reply:
To ensure this patch was tested on CI.
I stepped minor to override `builtin:bare-module` in addon cache.

Should i revert the bumps i did on other pr's as well?

Reply:
You should use a prerelease version for busting the cache, e.g. `4.8.3-0`. Release versions must only be tagged on `main`.